tests/test_parser.py

```diff
@@ -311,6 +311,32 @@ def test_if_empty_keys_raise_exception_with_empty_key(self,
             validate_object_keys(keys_list, creds)
         self.assertEqual('Key is empty', str(ke.exception))
 
+    @patch('requests.Session', return_value='session')
+    def test_login_with_params_uses_in_request(self, mock_session,
+                                               mock_headers,
+                                               user_credentials,
+                                               mock_request):
+        expected = {'username': 'username',
+                    'password': 'pass',
+                    'language': 'UA'}
+        login(username=expected['username'], password=expected['password'])
+        mock_request.assert_called_with(mock_session.return_value,
+                                        Urls.LOGIN.value,
+                                        expected,
+                                        mock_headers.return_value)
+
+    @patch('requests.Session', return_value='session')
```

Comment:
why do you set return_value here? It seems that it's not needed

Reply:
We need session object for call perform_post_request func, without this patch test fail

AssertionError: Expected call: perform_post_request('https://smarsy.ua/login.php?jsid=Login', {'username': 'username', 'password': 'pass', 'language': 'UA'}, {'h': '123'})
Actual call: perform_post_request(<requests.sessions.Session object at 0x10f31f1d0>, 'https://smarsy.ua/login.php?jsid=Login', {'username': 'username', 'password': 'pass', 'language': 'UA'}, {'h': '123'})

Reply:
I'm not talking about patch, I'm talking about return_value. the code works without it. I accidentally merged the request, but it would be nice to remove it as it is not used